client/me/help/main.jsx

```diff
@@ -108,9 +144,13 @@ const Help = React.createClass( {
 } );
 
 module.exports = reactRedux.connect(
-	( state ) => {
+	( state, ownProps ) => {
+		const isEmailVerified = currentUser.isCurrentUserEmailVerified( state );
+		const showCoursesTeaser = ownProps.isCoursesEnabled;
+
 		return {
-			isEmailVerified: currentUser.isCurrentUserEmailVerified( state ),
+			showCoursesTeaser,
+			isEmailVerified,
 		};
 	}
 )( Help );
```

Comment:
Everything in `ownProps` is automatically passed down to the component without needing to explicitly copy it in the `mapStateToProps`, so it's possible to just use `isCoursesEnabled` directly in the component.

Although I guess `showCoursesTeaser` might not necessarily `===ownProps.isCoursesEnabled` in the future?